frontend/src/components/SecureExchange/NewMessagePage.vue

```diff
@@ -64,20 +64,21 @@
                       </v-card-text>
                     </v-col>
                   </v-row>
-                  <v-row no-gutters>
-                    <v-col v-if="secureExchangeDocuments.length > 0 || secureExchangeStudents.length>0" class="d-flex px-0 pb-2">
-                      <v-chip id="documentChip" :class="['ma-1']" v-for="(document, index) in secureExchangeDocuments" :key="index" close @click:close="removeDocumentByIndex(index)">
+                  <v-row class="ml-6" no-gutters>
+                    <v-col cols="4" v-for="(document, index) in secureExchangeDocuments" :key="index" class="d-flex px-0 pb-2">
+                      <v-chip id="documentChip" :class="['ma-1']"   close @click:close="removeDocumentByIndex(index)">
                         <v-avatar left>
                           <v-icon>mdi-paperclip</v-icon>
                         </v-avatar>
-                        {{document.fileName}}</v-chip>
-                      <v-chip v-for="(secureExchangeStudent) in secureExchangeStudents" :key="secureExchangeStudent.studentID" :id="`studentChip-${secureExchangeStudent.studentID}`" :class="['studentChip', 'ma-1']" close @click:close="removeSecureExchangeStudentByID(secureExchangeStudent)">
+                        {{abbreviateFileName(document.fileName)}}</v-chip>
+                    </v-col>
+                    <v-col cols="4" v-for="(secureExchangeStudent) in secureExchangeStudents" :key="secureExchangeStudent.studentID" class="d-flex px-0 pb-2">
+                      <v-chip id="studentChip" :class="['ma-1']"  close @click:close="removeSecureExchangeStudentByID(secureExchangeStudent)">
```

Comment:
Hi @arcshiftsolutions,
I think we might have some id collisions for the studentChip.